apps/vs-code-designer/src/app/utils/binaries.ts

```diff
@@ -109,9 +109,12 @@ export async function validateAndInstallBinaries(context: IActionContext) {
         );
       } catch (error) {
         ext.outputChannel.appendLog(
-          localize('azureLogicApsBinariesError', 'Error in dependencies validation and installation: "{0}"...', error)
+          localize('azureLogicApsBinariesError', 'Error in dependencies validation and installation: "{0}"...', error?.message)
         );
-        context.telemetry.properties.dependenciesError = error;
+        vscode.window.showErrorMessage(
```

Comment:
Does this need to be localized?

Reply:
Ohhh yeah, I totally forgot about it. Thanks!